controls/V-72317.rb

```diff
@@ -63,4 +72,14 @@
 finding."
   tag "fix": "Remove all unapproved tunnels from the system, or document them with
 the ISSO."
+
+    @conn_grep_results = inspec.command("grep -i issue /etc/audit/audit.rules").stdout.split("\n")
```

Comment:
This doesn't seem right.

Reply:
Openswan doesn't exist in EL7, only `libreswan`

Reply:
Fixed that...sorry that was a really special testing mistake.